projects/online/monitor.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# trained model weights
+export AMPLFI_WEIGHTS=
+export AFRAME_WEIGHTS=
+# location where low latency data
+# is streamed, typically /dev/shm/kakfka
+export ONLINE_DATADIR=/dev/shm/kafka/
+# where results will be writen
+export AFRAME_ONLINE_OUTDIR=
+
+AFRAME_CONTAINER_ROOT=~/aframe/images/
+config=./config.yaml
+log_dir=
+mkdir -p $log_dir
+
+export CUDA_VISIBLE_DEVICES=2
+crash_count=0
+until apptainer run --nv --env "CUDA_VISIBLE_DEVICES=$CUDA_VISIBLE_DEVICES" $AFRAME_CONTAINER_ROOT/online.sif poetry run python online/cli.py --config $config 2>> $log_dir/monitoring.log; do
```

Comment:
We'll have to think a little bit about what running multiple models will look like. Currently, I run them from different branches, which requires having some code to switch branches in these monitoring scripts. But it might be simpler to just have multiple repos.

Reply:
Can you explain this a bit more not sure i follow